appengine/standard/ndb/schema_update/add_entities_handler.py

```diff
@@ -0,0 +1,32 @@
+import os
+import time
+
+import jinja2
+import webapp2
+
+import update_schema
+
+
+JINJA_ENVIRONMENT = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.dirname(__file__)),
+    extensions=['jinja2.ext.autoescape'],
+    autoescape=True)
+
+class AddEntitiesHandler(webapp2.RequestHandler):
+    def post(self):
+        author1 = self.request.get('author1', 'bob')
+        name1 = self.request.get('name1', 'bob')
+        author2 = self.request.get('author2', 'bob')
+        name2 = self.request.get('name2', 'bob')
+        update_schema.add_entity(author1, name1)
+        update_schema.add_entity(author2, name2)
+        time.sleep(1)
+        current_entities = update_schema.get_current_entities(False)
+        template_values = {
```

Comment:
Would probably make sense to make displaying the entities its own handler. Just have this redirect to the handler to display the entities.